packages/server/modules/auth/domain/logic.ts

```diff
@@ -1,11 +1,11 @@
-import type { UserInfo } from '@/modules/auth/domain/types'
+import type { UserinfoResponse } from 'openid-client'
```

Comment:
Lets not import an implementation type into our domain. Domain must not have implementation specific dependencies.

We're picking 3 string types.

Reply:
@gjedlicska Is this about the "domain" folder (which i'm still not clear on - its purpose or the rules around it)? Cause I'll just move this helper function to helpers then, cause this helper that uses this type only exists to assist w/ openid-client, it has no other meaning in the module and is definitely not of domain level importance.

Reply:
I've done just that - moved this to helpers